src/trainer.py

```diff
@@ -39,11 +38,8 @@ def train(self):
       dev_loss, dev_metrics = self.evaluate(loss_f, metrics)
       train_loss, train_metrics = self.evaluate(loss_f, metrics, mode='train')
 
-      wandb.log({
-        "loss/train": train_loss,
-        "loss/dev": dev_loss
-      }, step=epoch)
-
+      print(f"epoch {epoch}/{self.config.n_epochs}\ttrain loss: {mean_loss}\tdev loss: {dev_loss}")
```

Comment:
train_loss instead of mean_loss? mean_loss isn't defined, I suppose the code doesn't run now.

Reply:
nice catch, got mixed up